### DISCUSSION TITLE
Does getSession make a remote call when being used in a server side call?

### DISCUSSION THREAD

**Discussion by cubesnyc** [nextauthjs/next-auth#3700] [Help]
I not entirely sure what is happening under the hood here. When calling getSession from getStaticprops/getServerSideProps using jwt authentication, is a remote call to api/session being made? If yes, why is this the case?

**Comment by balazsorban44** [comment]
https://github.com/nextauthjs/next-auth/issues/1535#issuecomment-931785322

undocumented, but ready/available in v4. no guarantees yet, that's why it's not documented.
